qemu/tests/cfg/block_stream.cfg

```diff
@@ -108,3 +121,36 @@
                     when_streaming = "pause_job set_speed resume_job cancel start wait_for_finished"
                     Host_RHEL.m6:
                         when_streaming = "set_speed cancel start wait_for_finished"
+        - data_integrity:
+            type = block_stream_simple
+            before_start = "create_files stop"
+            after_finished = "resume verify_md5s"
+        - stop_cont:
+           type = block_stream_simple
+           before_start = "create_files"
+           when_streaming = "stop resume stop resume stop resume"
+           after_finished = "reboot verify_alive"
+        - with_installation:
+            type = block_stream_installation
+            wait_timeout = 36000
+            default_speed = 10240000
+            need_install = yes
+            start_vm = no
+            images = "stg"
+            image_name_stg = "images/block_stream_installation"
+            image_size_stg = 20G
+            boot_drive_stg = yes
+            source_image = "stg"
+            medium = cdrom
+            kernel = vmlinuz
+            initrd = initrd.img
+            force_create_image_stg = yes
+            image_aio = threads
+            unattended_delivery_method = cdrom
+            cdroms += " unattended"
```

Comment:
Add unattended cdrom only is not enough, if you want to use unattended iso ,   "cdrom_unattended"  need be defined for different guest.
and "unattended_delivery_method = cdrom" will replaced by "floppy" that defined in Windows.cfg in staf.
Currently,  it use answer.vfd,  not unattended.iso, so L149~150 can be removed.

If you don't want use floppy,  more cfg need be added to make it work.

Reply:
Refer the comments in  https://github.com/autotest/tp-qemu/pull/1233, this issue only exist for windows guest. 
All "with_installation" have this problem.

Reply:
@yanan-fu Thanks, as we discussed, let's keep it with floppy installation for now, and move to cdrom when the cdrom format issue fixed.